sse2neon.h

```diff
@@ -7592,14 +7592,22 @@ FORCE_INLINE int _mm_test_all_zeros(__m128i a, __m128i mask)
 // zero, otherwise set CF to 0. Return 1 if both the ZF and CF values are zero,
 // otherwise return 0.
 // https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#text=mm_test_mix_ones_zero
+// Note: Argument names may be wrong in the Intel intrinsics guide.
```

Comment:
One question about this. Why do you think the argument may be wrong. Just curious about the reason

Reply:
according to the Release notes:

on `01/24/2023` 
`Corrected parameter order of _mm_test_all_zeros and _mm_test_mix_ones_zeros.`

I don't know if this person still maintains this or not:
https://old.reddit.com/r/simd/comments/cck1kh/feedback_on_intel_intrinsics_guide/

I don't have a twitter account so I can't really track them down.

Reply:
got it!! Thank you for explaining!!!